install.php

```diff
@@ -24,11 +24,21 @@
             </ul>
         </li>
     </ol>
-
+    
+    <h2 class="hdr" id="quickstart">Upgrading to a new major release of macOS</h2>
+    
+    <ol>
+        <li>Install the latest version of <a href="<?php print $guide_url . '#installing.xcode'; ?>">Xcode and the Xcode Command Line Tools</a></li>
+        </li><a href="<?php print $trac_url . '/wiki/Migration#config'; ?>">Update your macports.conf</a> (if not default)</li>
+        <li>Reinstall MacPorts for your version of the Mac operation system</li>
```

Comment:
The word is "operating system", not "operation system" but let's just say "macOS", shall we?